### ISSUE TITLE
[MBTN_LEFT   cycle pause] causes video pausing when double-click (fullscreen toggle)

### ISSUE THREAD

**Issue by Mark-Joy** [mpvnet-player/mpv.net#685]
With [MBTN_LEFT   cycle pause] or [MBTN_LEFT   script-message-to mpvnet play-pause] in input.conf, video is cycle paused when left-mouse double-clicking (fullscreen toggle).

This can't be reproduced with mpv. In other words, this works as expected in mpv: left-mouse double-click only toggles fullscreen, and not causing video to pause/unpause.

**Additional context**
1. mpv.net version: v7.1.1.0
2. mpv version: 0.38.0
3. Windows version: Windows 10

**Comment by Sneakpeakcss** [comment]
Try putting `input-doubleclick-time=1` in `mpv.conf` and see if that fixes it for you.

**Comment by Mark-Joy** [comment]
> input-doubleclick-time=1

It still causes video to cycle pause.
Note that the fullscreen toggle feature works perfectly in original post even without `input-doubleclick-time=1`. It just that, after the fullscreen is toggled, the pausing is toggled as well.

**Comment by Sneakpeakcss** [comment]
I'm confused what's exactly the issue here. 

By default `mpv.net v7` doesn't bind anything under `MBTN_LEFT` and uses 
`MBTN_LEFT     ignore` same as default `mpv` config. When `MBTN_LEFT cycle pause` is used in `mpv` then it has the same behaviour of cycling pause/play during doubleclick (fullscreen).

**Comment by Mark-Joy** [comment]
> By default `mpv.net v7` doesn't bind anything under `MBTN_LEFT` 

https://github.com/mpvnet-player/mpv.net/blob/3f52c1255e68389c57b2ebdeb5f9bce09cdbf827/src/MpvNet/InputHelp.cs#L205
`MBTN_LEFT` is bind with `ignore` by default in `mpv.net`. In `mpv`, it doesn't.
And the only line in my input.conf is `MBTN_LEFT cycle pause`, for both

**Comment by Sneakpeakcss** [comment]
> (...) and uses MBTN_LEFT     ignore same as default mpv config.

> `MBTN_LEFT` is bind with `ignore` by default in `mpv.net`. In `mpv`, it isn't bind by anything by default.

[options-input-default-bindings / input-builtin-bindings](https://mpv.io/manual/master/#options-input-default-bindings)

[input.conf](https://github.com/mpv-player/mpv/blob/38741452480aeb402d9d02fe906fa7a7a3d6f5d7/etc/input.conf#L20-L31)

![image](https://github.com/mpvnet-player/mpv.net/assets/77424331/424844cf-e604-4300-90ee-291e7103d3ed)



What you're describing is literally default `mpv` behaviour, and using `MBTN_LEFT cycle pause` with a clean install of `mpv` behaves the same as you mentioned:

https://github.com/mpvnet-player/mpv.net/assets/77424331/d56bc0ce-a25b-424a-80e1-9a4da5b82d5a

**Comment by Mark-Joy** [comment]
Ok. Let me explain in a bit more details:
Suppose the video was in **`pause`** state before left-mouse double-clicking
 - In `mpv`: after double-clicking, the video is still on **`pause`** state
 - In `mpv.net`: after double-clicking, the video is changed to **`play`** state

Suppose the video was in **`play`** state before left-mouse double-clicking
 - In `mpv`: after double-clicking, the video is still on **`play`** state
 - In `mpv.net`: after double-clicking, the video is changed to **`pause`** state

In other words, in `mpv` the `play/pause` state is **not changed** after double-clicking. While in `mpv.net` the `play/pause` state is **toggle** after double-clicking

**Comment by Sneakpeakcss** [comment]
> Ok. Let me explain in a bit more details: Suppose the video was in **`pause`** state before left-mouse double-clicking
> 
>     * In `mpv`: after double-clicking, the video is still on **`pause`** state
> 
>     * In `mpv.net`: after double-clicking, the video is changed to **`play`** state
> 
> 
> Suppose the video was in **`play`** state before left-mouse double-clicking
> 
>     * In `mpv`: after double-clicking, the video is still on **`play`** state
> 
>     * In `mpv.net`: after double-clicking, the video is changed to **`pause`** state
> 
> 
> In other words, in `mpv` the `play/pause` state is **not changed** after double-clicking. While in `mpv.net` the `play/pause` state is **toggle** after double-clicking

This sounds like its related to [this](https://github.com/mpvnet-player/mpv.net/issues/634) then, and from the response in that issue it probably won't get fixed anytime soon.

**Comment by Mark-Joy** [comment]
https://github.com/mpvnet-player/mpv.net/blob/3f52c1255e68389c57b2ebdeb5f9bce09cdbf827/src/MpvNet/InputHelp.cs#L205
Well, I suppose there's no good reason for why the above code should be there. I mean, it is hard coded and furthermore it is bound to **`ignore`**

**Comment by wooshaq** [comment]
How does net changes mouse to regular mpv?
